frontend/src/Components/ReusableComponents/UserInfo.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from "react";
+import { useEffect } from "react";
+import Footer from "../Footer/Footer";
+import NavbarPanel from "../ReusableComponents/NavbarPanel";
+import ROUTES from "../Routes/ROUTES";
+import { getPersonalData } from "./ReusableFunctions";
+import "./UserInfo.css";
+
+const UserInfo = ({ userID }) => {
+   const [userData, setUserData] = useState();
+   const [errorMsg, setErrorMsg] = useState("");
+   const accessToken = localStorage.accessToken;
```

Comment:
I would prefer storing in memory token. This way if token changes for any reason, this component would not rerender. It is maybe no problem for your use case, but can be when app grows.